frontend/interfaces/EventTypes.ts

```diff
@@ -62,7 +73,8 @@ export const EmptyEventData: EventData = {
   image: "",
   eventTags: [],
   isActive: false,
-  attendees: [],
+  attendees: {},
```

Comment:
reorder so like isActive and isPrivate is together

Reply:
will fix